### PR TITLE
Student CV and photo

### PR DESCRIPTION
**Pull request by khdimon**
Added methods for get and update one student with image and CV files. Added validation for image and CV files name.

### CODE REVIEW

File: src/main/java/com/softserve/teamproject/entity/Student.java [hunk 1]
```diff
@@ -36,6 +36,22 @@
   @Column(name = "image")
   private byte[] image;
 
+  @Column(name = "image_name")
+  @Pattern(regexp = ".+(\\.jpg|\\.jpeg|\\.tiff|\\.png)",
+      message = "Sorry, you should use only allowed file types:"
```
Comment (thedikiy) [inline src/main/java/com/softserve/teamproject/entity/Student.java]: Please put into messages.propety

File: src/main/resources/liquibase-addCvChangeLog.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+  <changeSet id="add-column-CV" author="kholod">
+    <addColumn tableName="student">
+      <column name = "cv" type = "LONGBLOB">
```
Comment (thedikiy) [inline src/main/resources/liquibase-addCvChangeLog.xml]: We were talking that it's not good idea to store big objects in column. So it's better to make table, or store it in file.

File: src/main/java/com/softserve/teamproject/entity/Student.java (hunk 1 above)
Comment (khdimon) [inline src/main/java/com/softserve/teamproject/entity/Student.java]: Compilator says, here I can use only String constant for message, not message from property. Have you idea how to put messages.property in this place? I see in entities it doesn't use now.

File: src/main/resources/liquibase-addCvChangeLog.xml (hunk 2 above)
Comment (khdimon) [inline src/main/resources/liquibase-addCvChangeLog.xml]: We have another task for do it with User. I think here we must do it in same way. So, I propose to realise this together with User's image, maybe in next Sprint.

File: src/main/java/com/softserve/teamproject/entity/Student.java (hunk 1 above)
Comment (Crutovel) [inline src/main/java/com/softserve/teamproject/entity/Student.java]: i ll solve this problem in my pr with tests.
Do this as I did in CopyPasteScheduleWrapper.

File: src/main/java/com/softserve/teamproject/validation/impl/StudentValidatorImpl.java [hunk 3]
```diff
@@ -61,4 +61,15 @@ public void checkCoordinatorLocationToManipulateStudent(Group group, String user
       throw new AccessDeniedException("Coordinator can't add/edit students in the alien location.");
```
Comment (Crutovel) [inline src/main/java/com/softserve/teamproject/validation/impl/StudentValidatorImpl.java]: put in message.properties too

File: src/main/java/com/softserve/teamproject/entity/Student.java (hunk 1 above)
Comment (khdimon) [inline src/main/java/com/softserve/teamproject/entity/Student.java]: Done

**Comment by khdimon** [comment]
Dublicate branch

**Pull request closed without merging**